spec/features/contestant_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe Project, type: :feature do
+  describe "project show page" do
+    it "displays name and material and theme" do
+      recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+      furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+      news_chic = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+      boardfit = recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+      upholstery_tux = furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+      lit_fit = furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+      jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+      gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+      kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+      erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+      ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
+      ContestantProject.create(contestant_id: gretchen.id, project_id: news_chic.id)
+      ContestantProject.create(contestant_id: gretchen.id, project_id: upholstery_tux.id)
+      ContestantProject.create(contestant_id: kentaro.id, project_id: upholstery_tux.id)
+      ContestantProject.create(contestant_id: kentaro.id, project_id: boardfit.id)
+      ContestantProject.create(contestant_id: erin.id, project_id: boardfit.id)
+      
+      visit "/contestants"
+      expect(page).to have_content("Jay McCarroll")
+      expect(page).to have_content("Gretchen Jones")
+      expect(page).to have_content("Kentaro Kameyama")
+      expect(page).to have_content("Erin Robertson")
+      
+      expect(page).to have_content("Projects: News Chic")
+      expect(page).to have_content("Projects: News Chic, Upholstery Tux")
+    end
```

Comment:
We would like to see you use `within` blocks to help test where things are showing up on the page. It might also be best to break these into two tests - one that sees all the names of all contestants and one that sees a list of projects names under each contestant. 
Something like:
```ruby
it "I see a list of names of all the contestants" do
   visit "/contestants"

   expect(page).to have_content("Contestants")
   within "#contestants_list" do
      expect(page).to have_content(@jay.name)
      expect(page).to have_content(@gretchen.name)
      expect(page).to have_content(@kentaro.name)
      expect(page).to have_content(@erin.name)
   end
end   

it "And under each contestant's name I see a list of the projects (names) that they've been on" do
   visit "/contestants"

   within "#contestants_list" do
      within "#contestant_#{@jay.id}" do
        expect(page).to have_content("News Chic")
        expect(page).to_not have_content("Upholstery Tuxedo")
        expect(page).to_not have_content("Boardfit")
        expect(page).to_not have_content("Litfit")
      end

      within "#contestant_#{@gretchen.id}" do
        expect(page).to have_content("News Chic and Upholstery Tuxedo")
        expect(page).to_not have_content("Boardfit")
        expect(page).to_not have_content("Litfit")
      end
      .....
   end
end